plugins/spu2-x/src/Windows/Config.cpp

```diff
@@ -294,6 +305,19 @@ BOOL CALLBACK ConfigProc(HWND hWnd, UINT uMsg, WPARAM wParam, LPARAM lParam)
                     EndDialog(hWnd, 0);
                     break;
 
+                case IDC_OUTPUT:
+                    if (wmEvent == CBN_SELCHANGE) {
+                    const int module = (int)SendMessage(GetDlgItem(hWnd, IDC_OUTPUT), CB_GETCURSEL, 0, 0);
+                    bool IsConfigurable = module != 0;
+                    bool XAudio2Expansion = module == 1 || module == 4;
```

Comment:
Well, what does `1` and `4` refer to? They may currently refer to XAudio2 and PortAudio, but if, for instance, someone removes the DirectSound or WaveOut module or add a new Windows module, then that'll likely change.